manifests/r/RoyLFine/WinHTTPVPNNetworkExercisor/1.5.29.0/RoyLFine.WinHTTPVPNNetworkExercisor.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: RoyLFine.WinHTTPVPNNetworkExercisor
+PackageVersion: 1.5.29.0
+InstallerSwitches:
+  Silent: /VERYSILENT
+  Custom: /ALLUSERS
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://raw.githubusercontent.com/rlfine/MicrosoftStorePOCs/main/VPNUtilities/VpnExercisor/Installer/1.5.29.0/VPNExercisorSetup.exe
+  InstallerSha256: 4CD208A14D0BBCFBF1556D4A812D1B9AB0DE90D7DE042ED3B24E182CEC96C429
+  InstallerSwitches:
+    Silent: /VERYSILENT
+    Custom: /ALLUSERS

```

Comment:
```suggestion
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://raw.githubusercontent.com/rlfine/MicrosoftStorePOCs/main/VPNUtilities/VpnExercisor/Installer/1.5.29.0/VPNExercisorSetup.exe
  InstallerSha256: 4CD208A14D0BBCFBF1556D4A812D1B9AB0DE90D7DE042ED3B24E182CEC96C429
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://raw.githubusercontent.com/rlfine/MicrosoftStorePOCs/main/VPNUtilities/VpnExercisor/Installer/1.5.29.0/VPNExercisorSetup.exe
  InstallerSha256: 4CD208A14D0BBCFBF1556D4A812D1B9AB0DE90D7DE042ED3B24E182CEC96C429
  InstallerSwitches:
    Custom: /ALLUSERS
```

Reply:
NO - I don't want to force current user. an earlier PR comment was to remove the prompt for user to select, so I selected all users.

Reply:
Installer has two options to pick from, one user and one for all users.

User can choose which one they want by doing `winget install RoyLFine.WinHTTPVPNNetworkExercisor --scope machine` for All Users, or `winget install RoyLFine.WinHTTPVPNNetworkExercisor --scope user` for Current User.

Usually current user is preferred over all users because all users requires UAC.